DependencyInjection/DoctrineExtension.php

```diff
@@ -85,6 +87,14 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
     {
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('dbal.xml');
+        $chainLogger     = $container->getDefinition('doctrine.dbal.logger.chain');
+        $profilingLogger = new Reference('doctrine.dbal.logger');
```

Comment:
This variable name is really confusing, as `doctrine.dbal.logger` is *not* the profiling logger, but the logger logging your queries in the Symfony logs through PSR-3.

Reply:
Fixed.